storage/database/database_test.go

```diff
@@ -96,111 +99,104 @@ func newTestDynamoS3DB() (Database, func()) {
 	}
 }
 
-var test_values = []string{"a", "1251", "\x00123\x00"}
+var testValues = []string{"a", "1251", "\x00123\x00"}
```

Comment:
How about moving this to the inside of `TestPutGet`. Since our test code doesn't create a new database anymore, we better avoid the reuse of this variable in another test case in the future.